### ISSUE TITLE
Color Contrast Not High Enough

### ISSUE THREAD

**Issue by YuriDevAT** [EddieHubCommunity/EddieHubCommunity.github.io#302]
### Description

Elements must meet minimum color contrast ratio thresholds to meet the WCAG 2 AA .

- **4.5:1 for small text** - 14pt bold (19 CSS pixels), or
- **3:1 for large text** - 18pt (24 CSS pixels)

## Axe DevTools show that 6 elements do not meet color contrast on
<img width="494" alt="Axe DevTools results showing 6 elements do not meet color contrast" src="https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/assets/54622834/f158edf8-5641-43c5-9085-e9badb3d3043">

**Please find these elements and tips on how to improve in the next section 👇🏽** 

### Screenshots

I will show two ways that are valid for all 6 elements, using the first element as an example.

**TL;DR - How To Solve**
1. Increase the value to a minimum of `19px` **AND** set font-weight to a minimum of `700`.
2. Change text-color to a minimum of **#171717**.

💡 **Note:** The black you are using **#0F172A** for the intro (Open Source community aimed at encouraging and promoting communication) would work just fine.

---
### Element 1 - Link: **`<a><span>Get Started</span></a>`**
<img width="272" alt="Get started today - link" src="https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/assets/54622834/3c0b080c-bdc8-47f9-8079-162ae054ffa7">

#### How to solve

1. Increase the value to a minimum of `19px` AND set font-weight to a minimum of `700`.

<img width="377" alt="New values removes error" src="https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/assets/54622834/23139ff2-e153-4d40-b50e-a873417f5f61">

2. Change text-color to a minimum of **#171717**. 💡 **Note:** The black you are using **#0F172A** for the intro (Open Source community aimed at encouraging and promoting communication) would work just fine.
<img width="377" alt="New value removes error" src="https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/assets/54622834/affe74dd-d3da-490d-ac65-f6266f74fb70">

---
### Element 2 - Paragraph: `<p>Everyone is welcome to come and contribute to our open source projects.</p>`

<img width="693" alt="Paragraph showing error not meeting color contrast" src="https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/assets/54622834/76310644-7adb-48ab-9337-24b4a2d90e42">

---
### Element 3 - Paragraph: `<p>Community of inclusive Open Source people - Collaboration 1st, Code 2nd! Join our GitHub Org.</p>`

<img width="892" alt="Paragraph showing error not meeting color contrast" src="https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/assets/54622834/bf9779f0-dc07-4151-8649-47055c669349">

---
### Element 4 - Heading 3: `<h3>Discord</h3>`
<img width="717" alt="Heading showing error not meeting color contrast" src="https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/assets/54622834/d1ead736-7ad7-470b-ae7d-1b4126d4d312">

---
### Element 5 - Heading 3: `<h3>GitHub</h3>`
<img width="717" alt="Heading showing error not meeting color contrast" src="https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/assets/54622834/32d86050-616f-4cd6-98f6-521003d8bd8c">

---
### Element 6 - Button: `<button><span>Subscribe</span></button>`
<img width="542" alt="Button showing error not meeting color contrast" src="https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/assets/54622834/24b5ba18-035f-4194-b2d6-f646e42bfec9">

### Please find more information on color contrast issue and WCAG AA

https://dequeuniversity.com/rules/axe/4.7/color-contrast?application=AxeFirefox
https://www.w3.org/WAI/WCAG21/Techniques/general/G18.html
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

**Comment by xkrishguptaa** [comment]
Thank you yuri! Very helpful!

**Comment by xkrishguptaa** [comment]
- [x] Issue 1 (#313)
- [x] Issue 2 (#313)
- [ ] Issue 3
- [ ] Issue 4
- [ ] Issue 5
- [ ] Issue 6

**Comment by CodeTorso** [comment]
Now the get started button just looks odd. 

Can we have the background-color a bit more dark but the text as white?? 

This just looks more natural. I just feel like that, Not because I want to make my first PR in this project😄😄

<img width="183" alt="image" src="https://github.com/EddieHubCommunity/EddieHubCommunity.github.io/assets/129188028/211fb701-a639-4fd7-b648-e9fd11220953">
